### ISSUE TITLE
Kubedock high availability

### ISSUE THREAD

**Issue by pierrevillard** [joyrex2001/kubedock#96]
Hi,

Do you know if Kubedock support high availability ?
Does it's possible to run multiples replicas of kubedock pod behind a single kubernetes service for HA purpose ?

Thx

**Comment by joyrex2001** [comment]
No, that's not possible. kubedock has an internal database where it keeps track which containers (and other relevant resources) are created. When running multiple instances, this information is spread over these instances.
